### ISSUE TITLE
Page not found errors on HTML attribute reference page

### ISSUE THREAD

**Issue by RushilJalal** [mdn/content#27733]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

### What specific section or headline is this issue about?

Attribute names section

### What information was incorrect, unhelpful, or incomplete?

Several attribute names such as autoplay, autofocus, datetime had improper links, thus resulting in red squiggly lines.

### What did you expect to see?

Expected a link to the specific attribute page 

### Do you have any supporting links, references, or citations?

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus
seperate link for the autofocus page, which isn't linked in the attributes table.

### Do you have anything more you want to share?

![image](https://github.com/mdn/content/assets/54212357/6c5d955b-8076-406a-ab04-cf3f8af387e9)
![image](https://github.com/mdn/content/assets/54212357/c8101936-169c-4c9a-a1c2-0e5fb5a6f26c)

**Comment by Josh-Cena** [comment]
There are other broken links on this page so we can keep it open.

**Comment by RushilJalal** [comment]
Oh sorry. I thought the issue had been fixed

**Comment by estelle** [comment]
This is similar to 
https://github.com/mdn/content/issues/32229

**Comment by RushilJalal** [comment]
@estelle Is there anything I need to do?

**Comment by Josh-Cena** [comment]
No; we are systematically considering what to do with this page (and this area). We are most definitely going to make this an exhaustive table of all attributes, so we'll just remove most of them, making both this one (about non-existent links) and that one (about unlisted attributes) non-issues.

**Comment by Josh-Cena** [comment]
This page misses attributes and listed attributes don't have pages. We already have https://github.com/orgs/mdn/discussions/688 and https://github.com/mdn/content/issues/27733 to track the problem of incompleteness. I think we can simply close this one to avoid too many issues on the same problem.
